packages/cashier/src/Containers/cashier.jsx

```diff
@@ -90,13 +91,14 @@ const Cashier = ({
     if ((!is_logged_in && is_logging_in) || !is_account_setting_loaded) {
         return <Loading is_fullscreen />;
     }
+
+    const header_title = isMobile() ? selected_route.getTitle() : localize('Cashier');
+    const is_platform = platforms[platform] ? '' : header_title;
```

Comment:
`is_` prefix should be used only for boolean as per our JS guideline. Can you rename this variable?

Reply:
will do 👍🏼

Reply:
refactored to a function instead for readability 👍🏼  @balakrishna-binary